packages/beapi/src/player/Player.ts

```diff
@@ -63,6 +63,10 @@ export class Player {
     this.executeCommand(`tellraw @s {"rawtext":[{"text":"${message}"}]}`)
   }
 
+  public sendRawMessage(message: Array<object>): void {
```

Comment:
Please update the type to
```ts
Record<string, string>[]
```
[object is not type safe](https://stackoverflow.com/questions/64852546/dont-use-object-as-a-type)